pandas/tests/io/test_pickle.py

```diff
@@ -88,6 +88,13 @@ def compare_element(result, expected, typ):
 legacy_dirname = os.path.join(os.path.dirname(__file__), "data", "legacy_pickle")
 files = glob.glob(os.path.join(legacy_dirname, "*", "*.pickle"))
 
+if not len(files):
+    # Probably is a wheel build(no data files), so skip the entire module
```

Comment:
Shouldn't the `-no-strict-data-files` help to avoid having to skip?

Reply:
```
  ==================================== ERRORS ====================================
  _______________ ERROR collecting pandas/tests/io/test_pickle.py ________________
  Empty parameter set in 'test_pickles' at line 131
```
This was the error I was getting. It's a collection error, so unfortunately I can't skip the test with skipif.